util/cron/test-gpu-ex-cuda-12.interop.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# GPU native testing on a Cray EX (using none for CHPL_COMM)
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common.bash
+source $CWD/common-hpe-cray-ex.bash
+
+
+# We need 12.4 for the stream test because the CUDA driver on pinoak
+# only supports PTX for 12.4, untill the driver is updated, we need to
+# stick with 12.4 instead of 12.5
+module load cuda/12.4  # default is CUDA 12
+
+# We need cublas for the cublas interop test, but since we are using 12.4 above
+# pinoak doesn't have the cublas library for 12.4, so we need to use the cublas
+# from 12.5 (which is compatible across minor versions)
+# This can be removed once we use CUDA 12.5
+export CHPL_LIB_PATH="/opt/nvidia/hpc_sdk/Linux_x86_64/24.7/math_libs/lib64"
```

Comment:
I don't want this to become a wild goose chase, but I wonder if this should add `,$CHPL_LIB_PATH` in the end (if it is comma-separated) to avoid confusing issues down the road. I don't think that's very important, though.

Reply:
I will make this change (after checking if it's comma separated)

Reply:
Looks like it's `:` separated.